Controller/Main.as

```diff
@@ -161,9 +161,12 @@ void RefreshLeaderboard(){
 
         // Insert all entries in our temporary entry array
         for(uint i = 0; i< resp.positions.Length; i++){
-            if(resp.positions[i].time == -1){
+            if (false
+                || resp.positions[i].time == -1
+                || (i > 0 && resp.positions[i].time == resp.positions[i - 1].time)
+            )
                 continue;
-            }
+
```

Comment:
I assume the false is just to reorganize how that looks to align the other two conditions x), noice

What's the last condition preventing? (or to rephrase, what was the edge case you saw?)
I'm guessing you want to prevent the case where CM and WM have the same time ? (or sbville's medal and these two)

Reply:
Yeah, I tend to format conditionals with a `false` like that to make it hopefully more readable but mainly so I can easily comment entire lines to test things. In this case I think because the second condition is somewhat long, it would be hard to read being all on one line. Also yes, this is to hide the time if there are duplicates (CM/WM, CM/SB, WM/SB). Without this, I noticed the time would show in the window twice but both entries would say "Champion" and I couldn't find another way around it.

Reply:
Ideally I think I would want any duplicates to show so you can easily see if a CM is equal to a WM or whatever but with your logic I just couldn't find the way to do that

Reply:
Yeah, I guess it will stay as-is for now, and I'll open myself an issue to look at it later, since it would be better to see all medals yeah